packages/vision/src/index.js

```diff
@@ -1587,13 +1587,13 @@ Vision.formatFaceAnnotation_ = function(faceAnnotation) {
   };
 
   extend(formattedFaceAnnotation, Vision.convertToBoolean_(LIKELY, {
-    blurry: faceAnnotation.blurredLikelihood,
-    dark: faceAnnotation.underExposedLikelihood,
-    happy: faceAnnotation.joyLikelihood,
-    hat: faceAnnotation.headwearLikelihood,
-    mad: faceAnnotation.angerLikelihood,
-    sad: faceAnnotation.sorrowLikelihood,
-    surprised: faceAnnotation.surpriseLikelihood
+    anger: faceAnnotation.angerLikelihood,
+    blurred: faceAnnotation.blurredLikelihood,
+    headwear: faceAnnotation.headwearLikelihood,
+    joy: faceAnnotation.joyLikelihood,
+    sorrow: faceAnnotation.sorrowLikelihood,
+    surprise: faceAnnotation.surpriseLikelihood,
+    underExposed: faceAnnotation.underExposedLikelihood
```

Comment:
I feel like this could come up again in the future, could we just iterate over the `faceAnnotation` keys and chop off the `Likelihood` part?